### PR TITLE
Todo features

### PR DESCRIPTION
**Pull request by ScoobyDubeyDo**
- Added the todo features
- user can add, remove and check the todo
- the data is stored in Localstorage

### CODE REVIEW

File: src/screens/components/Todos/todos.module.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
+.todos-list {
+    border-radius: 0.5rem;
+    position: absolute;
+    background-color: rgba(0, 0, 0, 0.81);
+    /* min-height: 15rem; */
```
Comment (znjs) [inline src/screens/components/Todos/todos.module.css:5]: remove the commented code

File: src/screens/components/Todos/todos.module.css [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    max-height: 70vh;
+    width: 20rem;
+    bottom: 2rem;
+    right: 0;
+    display: flex;
+    overflow: auto;
+    justify-content: flex-start;
+    /* align-content: flex-end; */
```
Comment (znjs) [inline src/screens/components/Todos/todos.module.css:13]: can remove commented code

**Review by znjs — APPROVED** [review]
- Clean code
- Good naming conventions
- Nice logic applied

File: src/screens/components/Todos/todos.module.css (hunk 2 above)
Comment (ragini07) [inline src/screens/components/Todos/todos.module.css:13]: remove comments

File: src/screens/components/Todos/todos.module.css (hunk 1 above)
Comment (ragini07) [inline src/screens/components/Todos/todos.module.css:5]: remove comment

**Review by ragini07 — COMMENTED** [review]
Nice folder structure
Good naming convention

**Review by ragini07 — APPROVED** [review]

**Pull request merged**